packages/theme/src/cli/commands/theme/push.ts

```diff
@@ -139,33 +134,24 @@ export default class Push extends ThemeCommand {
 
   async run(): Promise<void> {
     const {flags} = await this.parse(Push)
-    const {path, force} = flags
-    const store = ensureThemeStore(flags)
-    const adminSession = await ensureAuthenticatedThemes(store, flags.password)
 
-    const workingDirectory = path ? resolvePath(path) : cwd()
-    if (!(await hasRequiredThemeDirectories(workingDirectory)) && !(await currentDirectoryConfirmed(force))) {
+    if (flags.password) {
```

Comment:
I am keeping the ruby implementation in this for now, given that this guard is for CLI users. 
This will NOT be exposed via the public package

This should be a simple removal at a later date 😄

Reply:
+100